src/order.rs

```diff
@@ -41,6 +42,13 @@ pub struct Order {
     pub compensatable_order_amount: u64,
     /// UUID of payment information that the order should be processed with.
     pub payment_information_id: Uuid,
+    /// Optional payment authorization information.
+    /// This field is not queriable with GraphQL.
+    #[graphql(skip)]
+    pub payment_authorization: Option<PaymentAuthorization>,
+    /// VAT number.
+    #[graphql(skip)]
```

Comment:
Does this `skip` thing mean that the VAT number is not queryable? Why shouldn't it be?

Reply:
because as we discussed with Steffen, it should be put on the invoice instead